app.js

```diff
@@ -26,7 +27,8 @@ exports.setup = function(callback) {
   }
 
   if (is_http_thread) {
-    app.listen(CONF.app.port);
+    http = http.createServer(app);
+    http.listen(CONF.app.port);
```

Comment:
just a sanity check: socket.io code sample at <http://socket.io/get-started/chat/> looks like:

```javascript
var app = require('express')();
var http = require('http').Server(app);
var io = require('socket.io')(http);
```

so they do `Server(app)` instead of `createServer(app)`, but createServer(app) returns a new instance of Server. Not sure if there is any material difference and/or if it matters?

Thanks